docs/examples/combined/calendar-agent.mdx

```diff
@@ -46,12 +51,16 @@ description: "This project is an example which uses Composio to seamlessly conve
         load_dotenv()
 
         # Initialize the language model
-        llm = ChatOpenAI(model="gpt-4o")
+        llm = ChatOpenAI(model="gpt-4")
```

Comment:
Good catch fixing the model name from `gpt-4o` to `gpt-4`. The incorrect model name would have caused API errors. Please ensure this is fixed consistently across all examples.